engine/admin/ansible.md

```diff
@@ -0,0 +1,72 @@
+---
+aliases:
+- /engine/articles//
+description: Installation and using Docker via Ansible
+keywords:
+- ansible, installation, usage, docker,  documentation
+menu:
+  main:
+    parent: engine_admin
+    weight: "11"
+title: Using Ansible
+---
+
+# Using Ansible
+
+> **Note**:
+> Please note this is a community contributed installation path.
+
+## Requirements
+
+To use this guide you'll need a working installation of
+[Ansible](https://www.ansible.com/) version 2.1.0 or later.
+
+Requirements on the host that will execute the module:  
+```
+python >= 2.6
+docker-py >= 1.7.0
+Docker API >= 1.20
+```
+
+## Installation
+
+The `docker_container` module is a core module, and will ship with 
+Ansible by default.
+
+## Usage
+
+Task example that pulls the latest version of the `nginx` image and 
+runs a container. Bind address and ports are in the example defined 
+as [a variable](https://docs.ansible.com/ansible/playbooks_variables.html).
+
+```
+---
+- name: nginx container
+  docker:
+    name: nginx
+    image: nginx
+    state: reloaded
+    ports:
+    - "{{ nginx_bind_address }}:{{ nginx_port }}:{{ nginx_port }}"
+    cap_drop: all
+    cap_add:
+      - setgid
+      - setuid
+    pull: always
+    restart_policy: on-failure
+    restart_policy_retry: 3
+		volumes:
+      - /some/nginx.conf:/etc/nginx/nginx.conf:ro
+  tags:
+    - docker_container
+    - nginx
+...
+```
+
+## Documentation
+
+The documentation for the `ansible_container` module is present at 
+https://docs.ansible.com/ansible/docker_container_module.html.
```

Comment:
Can you change this to include the link in the sentence, instead of dumping the URL at the end of the sentence.
Look at `/engine/admin/chef.md` for an example.